.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Test
+on: [push]
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.5, 2.6, 2.7, '3.0']
```

Comment:
What's the significance of quoting the 3.0 here?

Reply:
There's [this issue in the runner](https://github.com/actions/runner/issues/849) that for some reason specifically turns `3.0` into `3` if it's not quoted.